docs/pages/accounts/programmatic-access.mdx

```diff
@@ -17,7 +19,13 @@ Accounts can create Robot users to take actions on resources owned by the Accoun
 
 ## Access tokens usage
 
-You can use any tokens you have created to perform actions with the Expo CLI (other than signing in and out). To use tokens, you need to define an environment variable, like `EXPO_TOKEN="token"`, before running commands.
+You can use any tokens you have created to perform actions with the EAS CLI. To use tokens, you need to define an environment variable, like `EXPO_TOKEN="token"`, before running commands.
+Once you set the `EXPO_TOKEN` environment variable, you can run any EAS CLI command being authenticated with the token without a need to run the `eas login` command.
+The `eas login` command is only used for username and password authentication. The `EXPO_TOKEN` auth method takes precedence over the username and password if both are configured.
```

Comment:
```suggestion

Once you set the `EXPO_TOKEN` environment variable, you can run any EAS CLI command authenticated with the token without running the `eas login` command. The `eas login` command is only used for username and password authentication. The `EXPO_TOKEN` auth method takes precedence over the username and password if both are configured.
```

- Let's add a new line before this paragraph.
- The first sentence was a bit hard to read. Let's improve its readability to make it a bit more clear by using active voice.